.circleci/config.yml

```diff
@@ -15,17 +15,31 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
+          - v1-dependencies-build-{{ checksum "yarn.lock" }}
 
       - run: yarn install
 
       - save_cache:
           paths:
             - node_modules
-          key: v1-dependencies-{{ checksum "yarn.lock" }}
+          key: v1-dependencies-build-{{ checksum "yarn.lock" }}
 
       - run: make build RELEASE_BRANCH=${CIRCLE_BRANCH}
 
+      - run: rm -rf node_modules
```

Comment:
Is it ok to remove `node_modules` folder?

Reply:
The packages for runtime are installed in the following steps.